test/uk/gov/hmrc/cdsreimbursementclaim/services/ccs/CcsSubmissionServiceSpec.scala

```diff
@@ -60,6 +60,21 @@ class CcsSubmissionServiceSpec() extends AnyWordSpec with Matchers with MockFact
 
   implicit val hc: HeaderCarrier = HeaderCarrier()
 
+  def getHeaders(headerCarrier: HeaderCarrier): Seq[(String, String)] =
```

Comment:
The `headers` value they had before has been removed. So that's why I had to do this. I'm going to ask them about why they removed that. It was super-useful